tests/BreadcrumbTest.php

```diff
@@ -43,6 +43,24 @@ public function testConstructor(): void
         $this->assertEquals(microtime(true), $breadcrumb->getTimestamp());
     }
 
+    public function testConstructFromArray(): void
```

Comment:
As a naming convention for our tests we always use `test` + the name of the method being tested. In this case the correct name would be `testFromArray`